app/models/shipit/deploy_spec/lerna_discovery.rb

```diff
@@ -21,15 +23,25 @@ def discover_review_checklist
 
       def discover_lerna_checklist
         if lerna?
-          [%(
-            <strong>Don't forget version and tag before publishing!</strong>
-            You can do this with:<br/>
-            <pre>
-            lerna publish --skip-npm
-            && git add -A
-            && git push --follow-tags
-            </pre>
-          )]
+          if lerna_lerna >= LATEST_MAJOR_VERSION
+            [%(
+              <strong>Don't forget version and tag before publishing!</strong>
```

Comment:
nit: this can be de-duped with the `else` branch.

Reply:
done!